Muvr/MRSessionViewController.swift

```diff
@@ -6,18 +6,19 @@ import MuvrKit
 /// This class shows the exercises of the displayed session.
 /// To display a session, you must call the ``setSession(session:)`` method and provide a valid ``MRManagedExerciseSesssion``.
 ///
-class MRSessionViewController : UIViewController, UITableViewDataSource {
+class MRSessionViewController : UIViewController, UITableViewDataSource, UITableViewDelegate {
     
     @IBOutlet weak var tableView: UITableView!
     @IBOutlet weak var addLabelBtn: UIBarButtonItem!
     @IBOutlet weak var navbar: UINavigationBar!
     @IBOutlet var sessionBar: UINavigationItem!
+    @IBOutlet weak var uploadCSV: UIBarButtonItem!
     
     private var dataWaitingSpinner: UIBarButtonItem?
     
     // the displayed session
     private var session: MRManagedExerciseSession?
-    // indicates if the displayed session is active (i.e. not finished)
+    private var summaryExercises: [MRSummaryExercise] = []
```

Comment:
This would ideally go to an extension of the session. With this code (and the matching ``aggregateClassifiedExercises`` function), we are adding too much code to the controllers.